hs_app_netCDF/nc_functions/nc_meta.py

```diff
@@ -456,18 +456,22 @@ def extract_nc_data_variables_meta(nc_data_variables):
             'uint64': 'Unsigned Int64',
         }
 
-        try:
-            if isinstance(var_obj.datatype, netCDF4.CompoundType) or isinstance(var_obj.datatype, netCDF4.VLType):
-                nc_data_variables_meta[var_name]['type'] = 'User Defined Type'
-            elif var_obj.datatype.name in nc_data_type_dict.keys():
-                nc_data_variables_meta[var_name]['type'] = nc_data_type_dict[var_obj.datatype.name]
-            elif ('string' in var_obj.datatype.name) or ('unicode' in var_obj.datatype.name):
-                nc_data_variables_meta[var_name]['type'] = 'Char' if '8' in var_obj.datatype.name else 'String'
-            else:
+        if var_obj.dimensions:
+            try:
```

Comment:
I can't see the line of code within the try .. except block you expect to throw an exception.For that reason, I am suggesting there is no need for the try .. except